guides/workload-autoscaling/helmfile.yaml.gotmpl

```diff
@@ -0,0 +1,191 @@
+      - provider:
+          name: {{ .Environment.Values.provider.name }}
+      - istio:
+        {{ .Environment.Values.istio | toYaml | nindent 10 }}
+      - istio:
+          destinationRule:
+            host: {{ printf "gaie-%s-epp.%s.svc.cluster.local" $rn $ns | quote }}
+    {{- end }}
+    # Apply log level only in bench setting
+    {{- if (eq .Environment.Name "istioBench") }}
+      - inferenceExtension:
+          flags: 
+            {{ .Environment.Values.inferenceExtension.flags | toYaml | nindent 12 }}
+    {{- end }}
+    labels:
+      kind: inference-stack
+
+  - name: {{ printf "ms-%s" $rn | quote }}
+    namespace: {{ $ns }}
+    chart: llm-d-modelservice/llm-d-modelservice
+    version: v0.2.10
+    installed: true
+    needs:
+      - {{ printf "infra-%s" $rn | quote }}
+      - {{ printf "gaie-%s" $rn | quote }}
+    values:
+    {{- if eq .Environment.Name "gke_tpu" }}
+      - ms-inference-scheduling/values_tpu.yaml
+    {{- else if eq .Environment.Name "xpu" }}
+      - ms-inference-scheduling/values_xpu.yaml
+    {{- else if eq .Environment.Name "cpu" }}
+      - ms-inference-scheduling/values_cpu.yaml
+    {{- else if eq .Environment.Name "digitalocean" }}
+      - ms-inference-scheduling/digitalocean-values.yaml
+    {{- else if eq .Environment.Name "xpu" }}
+      - ms-inference-scheduling/values_xpu.yaml
+    {{- else }}
+      - ms-inference-scheduling/values.yaml
+    {{- end }}
+    {{- if (eq .Environment.Name "istioBench") }}
+      - routing: 
+          {{ .Environment.Values.routing | toYaml | nindent 10 }}
+    {{- end }}
+    set:
+    # apply release name derived values
+      - name: "routing.inferencePool.name"
+        value: {{ printf "gaie-%s" $rn | quote }}
+      - name: "routing.parentRefs[0].name"
+        value: {{ printf "infra-%s-inference-gateway" $rn | quote }}
+    {{- if or (eq .Environment.Name "gke") (eq .Environment.Name "gke_tpu") }}  
+      - name: "decode.monitoring.podmonitor.enabled"
+        value: false
+    {{- end }}
+    labels:
+      kind: inference-stack
+{{- if eq .Environment.Name "wva" }}
+  # Prometheus Adapter - Required for WVA to expose custom metrics
+  - name: prometheus-adapter
+    namespace: {{ env "MON_NS" | default "monitoring" }}
+    chart: prometheus-community/prometheus-adapter
+    version: 4.0.1
+    installed: true
+    needs:
+      - {{ printf "ms-%s" $rn | quote }}
+    labels:
+      kind: autoscaling
+    {{- if env "PROMETHEUS_ADAPTER_VALUES" }}
+    values:
+      - {{ env "PROMETHEUS_ADAPTER_VALUES" }}
+    {{- end }}
+  # Workload Variant Autoscaler - Dynamic autoscaling for llm-d
+  # Note: WVA chart can be installed from a local path or OCI registry
+  # Set WVA_CHART_PATH to use a local chart (e.g., after cloning the repo)
+  # Otherwise, it will attempt to use the OCI registry path
```

Comment:
This feels like it should be structured as a dependency chart of their helm chart rather than an entry in our helmfile, I will need to look into this

Reply:
how would you indicate a pure dependency and not including the chart here?

Reply:
done